### PR TITLE
Adding mr lang

### PR DESCRIPTION
**Pull request by ShrinathPawar**
- **This PR is about about updateding files** 

- **[Worked on this issue](https://github.com/novuhq/novu/issues/1367) **

### CODE REVIEW

**Comment by ShrinathPawar** [comment]
I should have ask if you can assign me this issue to be resolved before working on this ?

**Comment by scopsy** [comment]
@ShrinathPawar could you please add the language to our docs, and also what MR stands for? @BiswaViraj could you take a look?

File: packages/notification-center/src/i18n/lang.ts [hunk 1]
```diff
@@ -50,6 +51,7 @@ export const TRANSLATIONS: Record<I18NLanguage, ITranslationEntry> = {
   or: OR,
   sa: SA,
   ne: NE,
+  mr: Mr,
```
Comment (p-fernandez) [inline packages/notification-center/src/i18n/lang.ts:54]:
```suggestion
  mr: MR,
```
This was a bug.

File: packages/notification-center/src/i18n/languages/mr.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const HR: ITranslationEntry = {
+  translations: {
+    notifications: 'सुचना',
+    markAllAsRead: 'सर्व वाचल्याप्रमाणे चिन्हांकित करा',
+    poweredBy: 'द्वारा समर्थित',
+    settings: 'सेटिंग्स',
```
Comment (p-fernandez) [inline packages/notification-center/src/i18n/languages/mr.ts:8]: Google Translate gives me some inconsistencies regarding capitalisation, could you review them please?

**Comment by ShrinathPawar** [comment]
@p-fernandez have a look to this last PR

File: packages/notification-center/src/i18n/languages/mr.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const HR: ITranslationEntry = {
```
Comment (scopsy) [inline packages/notification-center/src/i18n/languages/mr.ts:3]:
```suggestion
export const MR: ITranslationEntry = {
```

Just noticed that the export is wrong here :)

**Review by BiswaViraj — APPROVED** [review]

**Comment by ainouzgali** [comment]
Hi @ShrinathPawar , could you please merge the conflicts here? And then we'll merge🚀

**Comment by ShrinathPawar** [comment]
> Hi @ShrinathPawar , could you please merge the conflicts here? And then we'll merge🚀

I did it.

**Comment by Asmit2952** [comment]
Hey @ShrinathPawar I had opened this issue on the repository and I was working on it. Here: https://github.com/novuhq/novu/issues/1367

Can you please find another issue to work on?

**Comment by scopsy** [comment]
Hi @ShrinathPawar, seems like this issues was created by another contributor and already was contributed by him.

Please ask for assignment in the future before working on something, just to make sure we don't have duplicate entries. Will have to close this issue for now 🙏

**Pull request closed without merging**